packages/api-database/source/models/block.ts

```diff
@@ -91,5 +91,5 @@ export class Block {
 		// TODO: length depends on signature size...
 		// length: 256,
 	})
-	public readonly blockSignature!: string;
+	public readonly signature!: string;
```

Comment:
Block signature was removed from block instance. Aggregated signature is now part of committed block.

Reply:
Signature is aggregated signature of committed block, so original comment is irrelevant.